examples/tutorials/02_pipeline_operations.ipynb

```diff
@@ -15,7 +15,6 @@
    "source": [
```

Comment:
<p>I suggest avoid using positional arguments in public notebooks.</p><p> </p><p>Argument <code>shuffle</code> is also redundant here.</p><p> </p><p>Suggestion:</p><p> </p><p><code>pipeline.run(batch_size=1, n_iters=10)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>